src/main/java/ch/njol/skript/config/Node.java

```diff
@@ -238,39 +243,39 @@ private static SplitLineState update(char c, SplitLineState state, SplitLineStat
 			return state;
 		}
 	}
-	
+
 	static void handleNodeStackOverflow(StackOverflowError e, String line) {
 		Node n = SkriptLogger.getNode();
-		SkriptLogger.setNode(null); // Avoid duplicating the which node error occurred in parentheses on every error message
-		
+		SkriptLogger.setNode(null); // Avoid duplicating the which node error occurred in paranthesis on every error message
```

Comment:
```suggestion
		SkriptLogger.setNode(null); // Avoid duplicating the which node error occurred in parentheses on every error message
```
I believe the original usage was correct